### ISSUE TITLE
Cannot install tinycudann unfortunately W11

### ISSUE THREAD

**Issue by DiamondGlassDrill** [princeton-computational-imaging/NSF#3]
Thanks for the amazing work.
I cannot unfortunately install tinycudann due to the error bellow.
If someone on W11 comes across the same error or/and has a solution, please let me know.

CUDA 11.8 is installed
Ninja the latest

/usr/bin/link: extra operand '/LTCG'
Try '/usr/bin/link --help' for more information.
error: command 'C:\\msys64\\usr\\bin\\link.exe' failed with exit code 1

Thanks in advance.

**Comment by Ilya-Muromets** [comment]
Yeah, sorry, haven't tested anything for Windows as I do all my work on a linux machine. I have heard of people getting Ubuntu + CUDA running through [https://developer.nvidia.com/cuda/wsl](https://developer.nvidia.com/cuda/wsl) though, just as another potential option.
